test/unit/gateways/opp_test.rb

```diff
@@ -203,6 +226,14 @@ def failed_response(type, id, code='100.100.101')
     )
   end
 
+  def failed_store_response(id, code='100.100.101')
+    OppMockResponse.new(400,
+      JSON.generate({'id' => id, 'result' => {'code' => code, "description" => 'invalid creditcard, bank account number or bank name'},
```

Comment:
Now Rubocop is complaining about this `"description"` using double-quotes instead of single-quotes.

```
C: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)
      JSON.generate({'id' => id, 'result' => {'code' => code, "description" => 'invalid creditcard, bank account number or bank name'},
                                                              ^^^^^^^^^^^^^
```